libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -99,49 +99,48 @@ proc replenishFanout(g: GossipSub, topic: string) =
   trace "fanout replenished with peers", peers = g.fanout.peers(topic)
 
 proc rebalanceMesh(g: GossipSub, topic: string) {.async.} =
+  logScope:
+    topic
+
   trace "about to rebalance mesh"
+
   # create a mesh topic that we're subscribing to
 
   var
     grafts, prunes: seq[PubSubPeer]
 
   if g.mesh.peers(topic) < GossipSubDlo:
-    trace "replenishing mesh", topic, peers = g.mesh.peers(topic)
-    # replenish the mesh if we're below GossipSubDlo
-    var newPeers = toSeq(
+    trace "replenishing mesh", peers = g.mesh.peers(topic)
+    # replenish the mesh if we're below Dlo
+    grafts = toSeq(
       g.gossipsub.getOrDefault(topic, initHashSet[PubSubPeer]()) -
       g.mesh.getOrDefault(topic, initHashSet[PubSubPeer]())
     )
 
     logScope:
-      topic = topic
       meshPeers = g.mesh.peers(topic)
-      newPeers = newPeers.len
+      grafts = grafts.len
```

Comment:
what is this doing?

Reply:
printing number of graft candidates I guess

Reply:
I think this is resetting the seq's size, but it's setting it to itself, so it's a noop, look fishy. I know it was there before you got to it, but seems wrong either way.

Reply:
oh, it's a logscope, nevermind.